<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>userinfo</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QLineEdit" name="ID"/>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="batch"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>批号</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="unpasstype"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>生产日期</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>不良品名称</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="unpassname"/>
        </item>
        <item row="0" column="6">
         <widget class="QLineEdit" name="prodate"/>
        </item>
        <item row="1" column="6">
         <widget class="QLineEdit" name="unpassqty"/>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>数量Kg</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>不良品种类</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="1" column="0">
            <layout class="QFormLayout" name="formLayout_2"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>原因分析</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QTextEdit" name="result"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>纠正措施</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QTextEdit" name="correctiveaciton"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>预防措施</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QTextEdit" name="precaution"/>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="check_selfprev">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>自审核</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>不良品描述</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="describe"/>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="lbl_writerinfo">
             <property name="text">
              <string>writer_info</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>模糊查找</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_11"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_search">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/search.ico</normaloff>icons/search.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_caseto">
              <property name="text">
               <string>Case To</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tbl_unpass"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单(M)</string>
    </property>
    <addaction name="act_changepwd"/>
    <addaction name="act_chguser"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_changepwd">
   <property name="icon">
    <iconset>
     <normaloff>icons/password32px.ico</normaloff>icons/password32px.ico</iconset>
   </property>
   <property name="text">
    <string>更改密码</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="act_chguser">
   <property name="icon">
    <iconset>
     <normaloff>icons/user32px.ico</normaloff>icons/user32px.ico</iconset>
   </property>
   <property name="text">
    <string>更换用户</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset>
     <normaloff>icons/quit128px.ico</normaloff>icons/quit128px.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>icons/dir.png</normaloff>icons/dir.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
